packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -257,9 +257,11 @@ export class ResourceService {
   async createService(
     args: CreateOneResourceArgs,
     user: User,
-    wizardType: string = null
+    wizardType: string = null,
```

Comment:
it's many args, I would think to convert it to object instead of parameters

Reply:
For 2 optional params???? @barshimi